Neato config flow (#10356)
home-assistant/home-assistant.github.io: source/_components/neato.markdown

The `neato` integration allows you to control your [Neato Botvac Connected Robots](https://www.neatorobotics.com/robot-vacuum/botvac-connected-series/).

To activate `neato` in your installation, you can set it up from the integration screen or add it to your `configuration.yaml` file.

## Setup the integration via the integrations screen

Menu: *Configuration* -> *Integrations*

> Press on **Neato** and configure the integration. You will need to enter your username and password and whether you are using a Neato or Vorwerk device.

Reviewer: ```suggestion
Search for or select **Neato** from the list and configure the integration. You will need to enter your username and password and whether you are using a Neato or Vorwerk device.
```

Author: Done, thank you
